.github/workflows/CI.yml

```diff
@@ -84,6 +84,21 @@ jobs:
       run: |
         CHPL_HOME=$PWD make run-dyno-linters
 
+  make_check_llvm_none:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/${{ github.repository_owner }}/chapel-github-ci:latest
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.github_token }}
+    steps:
+    - uses: actions/checkout@v3
+    - name: make check_llvm_none
+      run: |
+       CHPL_LLVM=none ./util/printchplenv --all make make check
```

Comment:
I looked closer at the non-llvm-none version of this job and realized this is not running the same command -- I assume it is meant to be

Reply:
Hi Anna, This PR is still work in progress.
Just tried to push to to github to understand the behavior.

Reply:
Oops, sorry